_posts/2020-02-12-myft-machine-learning.html

```diff
@@ -0,0 +1,105 @@
+
+<h2>Suggestr</h2>
+
+<p>We built Suggestr as an endpoint that takes one or more FT topics as input and can suggest other topics of interest. We also included the correlation service, which also powered the correlations game (<a href="https://labs.ft.com/games/2017/09/18/make-connections.html">https://labs.ft.com/games/2017/09/18/make-connections.html</a>), so that we could find topics that appear in the same cluster as the input topic(s), exclude topics that directly correlate and then rank the remaining with their follows/unfollow ratio. This way we suggest topics that don't appear in the same articles as the input topic but have been determined (by ML analysis of user interactions) to be of similar interest.</p>
+
+<p>So Suggestr was born:  <a href="https://github.com/ftlabs/suggestr">https://github.com/ftlabs/suggestr</a></p>
+
+{% include figure.html src="/assets/uploads/2020/02/myft_ml_suggestr_output.png" description="Example screenshot of Suggestr's output when providing Blockchain as an input topic. Core results are in the results object, the rest of the output shows the steps taken to get the results, as not to create too much of a black box recommendation." width="80%" %}
+
+<h2>Suggestr's mission</h2>
+
+<p>Using topic cluster analysis, topic correlations data and topic follow/unfollow ratios - can we suggest retrieve topics to follow (or retrieve articles) for based on any given topic(s)?</p>
+
+<p>The first prototype of Suggestr works well but we'd like to refine and add to the ranking filters before user (reader) testing can begin.</p>
+
+<h2>What did we learn?</h2>
+
+<p>All those ML & AI experts saying data prep is the largest time consumption percentage of the project weren't kidding. However all that time familiarised us with the data and allowed us to build effective tools to minimise time spent if we revisit that data.</p>
+
+<p>If we don't record the data we have in a documented or understandable way that priceless data is basically number garbage. You need to be able to trust your data, to trust your analysis for your users to trust your output. So let your use of the recorded data shape the future recorded data, a cycle that will continue to improve your data (and projects) into the future.</p>
+
+<p>
+<i>"Every company has messy data, and even the best AI companies are not fully satisfied with their data."</i>
+<br />
+<b>Andrew Ng</b>
+</p>
+
+<p>It is difficult to generate recommendations based on a few metrics, you'll want as many as you can to shape the best suggestion. In the end there may be no good suggestions right now, and it's okay to say that rather than making a bad/false suggestion and losing the trust of a user.</p>
+
+<p>Our bumbling around in the data helped other teams understand how tricky navigating this data can be without help. Efforts are already underway to clarify and clean the data for everyone's use.</p>
```

Comment:
navigating this data = navigating it, to avoid the repetition of "data"

Reply:
Done